From 2d8c2049c7caad19e6cacccea1bc8962ac481f5a Mon Sep 17 00:00:00 2001
Date: Wed, 17 Jun 2020 10:10:35 +0000
Subject: [PATCH] 8223667: ASAN build broken

Summary: <asan>: <Remove quote from ASAN_OPTIONS variable in make/autoconf/spec.gmk.in>
LLT: N/A
Bug url: https://bugs.openjdk.java.net/browse/JDK-8223667
---
 make/autoconf/spec.gmk.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/make/autoconf/spec.gmk.in b/make/autoconf/spec.gmk.in
index 9473481bc..38fa47d9f 100644
--- a/make/autoconf/spec.gmk.in
+++ b/make/autoconf/spec.gmk.in
@@ -382,7 +382,7 @@ GCOV_ENABLED=@GCOV_ENABLED@
 export ASAN_ENABLED:=@ASAN_ENABLED@
 export DEVKIT_LIB_DIR:=@DEVKIT_LIB_DIR@
 ifeq ($(ASAN_ENABLED), yes)
-  export ASAN_OPTIONS="handle_segv=0 detect_leaks=0"
+  export ASAN_OPTIONS=handle_segv=0 detect_leaks=0
   ifneq ($(DEVKIT_LIB_DIR),)
     export LD_LIBRARY_PATH:=$(LD_LIBRARY_PATH):$(DEVKIT_LIB_DIR)
   endif
-- 
1.8.3.1
